### PR TITLE
New Images and Mainfest.json

### PR DESCRIPTION
**Pull request by EinfachFloTV**
## Type of change

Images and Mainfest.json

### CODE REVIEW

File: minecraft_servers/craftportal/manifest.json [hunk 1]
```diff
@@ -3,11 +3,11 @@
   "nice_name": "Craftportal",
   "direct_ip": "craftportal.net",
   "supported_languages": [
-    "de"
+    "de", "en"
   ],
   "social": {
-      "discord":"https://discord.gg/7F6MbnUSEp",
-      "teamspeak":"ts.craftportal.net",
+      "web": "https://craftportal.net",
```
Comment (DoJapHD) [inline minecraft_servers/craftportal/manifest.json:9]: Not reachable

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/craftportal/manifest.json [hunk 2]
```diff
@@ -3,11 +3,11 @@
   "nice_name": "Craftportal",
   "direct_ip": "craftportal.net",
   "supported_languages": [
-    "de"
+    "de", "en"
   ],
   "social": {
-      "discord":"https://discord.gg/7F6MbnUSEp",
-      "teamspeak":"ts.craftportal.net",
+      "web": "https://www.craftportal.net",
```
Comment (DoJapHD) [inline minecraft_servers/craftportal/manifest.json:9]: https://i.imgur.com/hjhON0F.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/craftportal/manifest.json (hunk 2 above)
Comment (mrcraftreal) [reply minecraft_servers/craftportal/manifest.json:9]: https://postimg.cc/3d5vY56r

Comment (DoJapHD) [reply minecraft_servers/craftportal/manifest.json:9]: https://i.imgur.com/b5b6Niw.png

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by mrcraftreal** [comment]
Our website should now be working again and I have removed Teamspeak.

File: minecraft_servers/craftportal/manifest.json (hunk 2 above)
Comment (DoJapHD) [inline minecraft_servers/craftportal/manifest.json:9]: If website is in maintenance mode, there's no real meaning behind having it in here
If the website will be in maintenance for longer, remove

**Review by DoJapHD — CHANGES_REQUESTED** [review]

**Comment by mrcraftreal** [comment]
Okay. Can you please delete Teamspeak?

**Comment by DoJapHD** [comment]
The PR owner can do that, I won't.

**Comment by DoJapHD** [comment]
Inactive PR

**Pull request closed without merging**